### PR TITLE
community/libwmf: resolve collision with ghostscript-fonts

### PR DESCRIPTION
**Pull request by andypost**
Fixes https://bugs.alpinelinux.org/issues/8619

### CODE REVIEW

**Comment by andypost** [comment]
Now bump let's see how tests will work with https://github.com/alpinelinux/aports/pull/3519

**Comment by andypost** [comment]
Looks this both changes can't go in one PR cos `php7-gmagick` loads `libwmf` from repo which brings collide with new dependency

File: community/libwmf/APKBUILD [hunk 1]
```diff
@@ -33,6 +34,8 @@ build() {
 package() {
 	cd "$builddir"
 	make -j1 DESTDIR="$pkgdir" install || return 1
```
Comment (TBK) [inline community/libwmf/APKBUILD]: Remember to remove `|| return 1`

File: community/libwmf/APKBUILD [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
 pkgver=0.2.8.4
-pkgrel=3
+pkgrel=4
 pkgdesc="A library for reading vector images in Microsoft's native Windows Metafile Format (WMF)."
 url="http://wvware.sourceforge.net/libwmf.html"
 arch="all"
 license="LGPL"
+depends="ghostscript-fonts"
 makedepends="libpng-dev>=1.4.0 freetype-dev jpeg-dev>=8 expat-dev"
 install=
```
Comment (TBK) [inline community/libwmf/APKBUILD]: Since `install=` is not used it can be removed.

**Pull request closed without merging**